fe/fe-core/src/main/java/com/starrocks/catalog/ColocateTableIndex.java

```diff
@@ -498,12 +498,16 @@ public void clear() {
         }
     }
 
-    protected boolean validDbIdAndTableId(long dbId, long tableId) {
+    private String getTableName(long dbId, long tableId) {
         Database database = GlobalStateMgr.getCurrentState().getDb(dbId);
         if (database == null) {
-            return false;
+            return null;
+        }
+        Table table = database.getTable(tableId);
+        if (table == null) {
+            return null;
         }
-        return database.getTable(tableId) != null;
+        return database.getOriginName() + "." + table.getName();
```

Comment:
I suggest you to add an extra column called `DatabaseName` since all  tables of a ColocatedGroup belong to the same Database. Otherwise `TableName` can be really long with DatabaseName.